internal/dev_server/sdk/store_facade.go

```diff
@@ -0,0 +1,42 @@
+package sdk
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/launchdarkly/ldcli/internal/dev_server/model"
+	"github.com/pkg/errors"
+)
+
+// WriteError writes out a given error if it's known or panics if it isn't.
+// Two assumptions it's making
+//   - a panic handling middleware is in use
+//   - This is in the context of flag delivery which has pretty consistent semantics for what's an error across handlers.
```

Comment:
Is this reasoning valid for the cli? Or is it sort of defining a new / the convention here?

Reply:
Not valid, in general for CLI commands. This is just for the flag delivery endpoints provided by the dev server.